api/lib/application/error-manager.js

```diff
@@ -131,6 +131,9 @@ function _mapToHttpError(error) {
   if (error instanceof DomainErrors.UserNotFoundError) {
     return new HttpErrors.NotFoundError(error.message);
   }
+  if (error instanceof DomainErrors.UserShouldChangePasswordError) {
+    return new HttpErrors.PasswordExpiredError(error.message);
```

Comment:
J'ai une petite incompréhension avec ce choix de nom, comme si PasswordExpired signifiait qu'il existe une date d'expiration.

Est-ce que `PasswordShouldChange` ne serait pas plus adapté ?

Reply:
Je corrige

Reply:
Plutôt **Must**Change non ?

Reply:
Christophe et moi notons bien qu'il y a des axes d'amélioration sur le nommage. Nous avons déjà mené une action dans ce sens pour les futurs sujets (CGU).
Quand à ce ticket, le rework a déjà eu lieu une fois.  Nous proposons de ne pas le faire, à décider en point tech.

Reply:
Le hasard fait bien les choses: je suyis tombé sur [ça](http://www.faqs.org/rfcs/rfc2119.html) ce matin

Reply:
Oui c'est cette norme que j'ai en tête quand j'entends should/must 😉